mathics/core/characters.py

```diff
@@ -12,7 +12,7 @@
 
 # Load the data on characters
 with open(os.path.join(ROOT_DIR, "data/characters.yml"), "r") as f:
-    _CHAR_DATA = yaml.load(f)
+    _CHAR_DATA = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
As suggested before, time this,  and if this is slow speed it up 
by doing the conversion and preprocessing once at install time.

Reply:
Ohh, I see. You're idea is to load the YAML at install time and pickle the dictionary or something?

Reply:
https://stackoverflow.com/questions/18517949/what-is-faster-loading-a-pickled-dictionary-object-or-loading-a-json-file-to  

* pickle
* cPickle
* json
* simplejson
* ujson
* yajl

Gives timing from slowest (pickle) to fastest (yajl). since the code is there, we should try running it on our particular data to see which works.

Reply:
Ok, I'm running the tests now. Apparently `cpickle` isn't used anymore though (see https://stackoverflow.com/questions/56562929/modulenotfounderror-no-module-named-cpickle-on-python-3-7-3/56563226#56563226)